examples/app-example-androidjunitrunner/app/src/androidTest/java/com/example/screenshots/MainActivityTest.java

```diff
@@ -21,21 +21,24 @@
 public class MainActivityTest {
 
     @Rule
-    public ActivityTestRule<MainActivity> mActivityTestRule = new ActivityTestRule<>(MainActivity.class);
+    public ActivityTestRule<MainActivity> mActivityTestRule = new ActivityTestRule<>(MainActivity.class, true, false);
```

Comment:
Why the change here?

Reply:
We need this change if we want to launch the activity the way we do it now. If we do as before, mainActivityTestSettingsOpen is not a deterministic test.